internal/testutils/admock/ldb/__init__.py

```diff
@@ -119,7 +119,7 @@ def addGPO(self, gpo):
         self.gPLink = [gPLink]
 
     def addAccount(self, accountName):
-        Account(accountName, self)
+        Account(accountName.lower(), self)
```

Comment:
ditto here, let’s lowercase every data we have in the mock rather than relying on the mock implementation.